Content.Shared/Silicons/StationAi/SharedStationAiSystem.Airlock.cs

```diff
@@ -1,25 +1,103 @@
 using Content.Shared.Doors.Components;
 using Robust.Shared.Serialization;
+using Content.Shared.Electrocution;
+using Content.Shared.Popups;
+using Content.Shared.Power.Components;
+using Robust.Shared.Audio;
 
 namespace Content.Shared.Silicons.StationAi;
 
 public abstract partial class SharedStationAiSystem
 {
+    private static readonly SoundPathSpecifier AirlockOverchargeDisabled = new("/Audio/Machines/airlock_overcharge_on.ogg");
```

Comment:
Couldnt you put those in the component or would that break things?

Reply:
Not sure where to put this. Maybe maint will propose good ideas. AirlockComponent is not used in this code path, ElectrifyComponent - well, fine but its still wierd... its only hear-able to AI.

Reply:
If it's only hear-able to AI, maybe put specifiers into AI Core or Held? It would allow change audio paths in prototypes...

Reply:
@metalgearsloth  what do you think about moving this to HeldComponent example? sounds a bit sus

Reply:
This should be wherever stuff is electrified, also don't call it "overcharge" because it's just confusing what it's for.